docs/sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,55 @@
 Version history
 ===============
 
+5.8.0 (2018 February 20)
+------------------------
+
+File format fixes and improvements:
+
+* cellSens VSI
+   - added support for lossless JPEG compression
+* Imspector OBF
+   - improved the parsing of OBF files with embedded OME-XML metadata (thanks to Bjoern Thiel)
+* Ionpath MIBI
```

Comment:
This is an entirely new format, isn't it? I think we should list it separately at the top as a new format as per 5.5.0

Reply:
Makes sense. In which case we should also move Vectra QPTIFF there.